experimental/bindings/java/build.gradle

```diff
@@ -54,8 +58,30 @@ android {
 
     externalNativeBuild {
         cmake {
-            version "3.13.0+"
+            version "3.18.4+"
             path "../../../CMakeLists.txt"
         }
     }
 }
+
+// Task to cmake configure the host
+task cmakeConfigureHost(type: Exec) {
+    doFirst {
+        println "Configuring host tools with cmake..."
```

Comment:
If you're just building host tools, you could set more flags to prune the number of targets built.

https://github.com/google/iree/blob/c67112d825feb8a8c820756f24ce6200028d2137/build_tools/cmake/build_android.sh#L53-L59

In particular, set `-DIREE_BUILD_TESTS=OFF` so tests are not built or installed (those compile models and can take up to several minutes depending on the machine/config).